discord/sticker.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+
+"""
+The MIT License (MIT)
```

Comment:
The license header is not in the same exact format as the other files. This one's inconsistent with the rest of the library.